_layouts/product-get-started.html.slim

```diff
@@ -71,6 +70,10 @@ ol
     h3.divider#ExtraSection #{page._extra_section_title}
   = page._extra_section
 
+h3.divider#bpm_suite_starter_kit JBoss BPM Suite Starter Kit
```

Comment:
This layout file is used by all products. So by adding these here, you have added the content to all products' get-started pages.